olp-cpp-sdk-authentication/src/AuthenticationCredentials.cpp

```diff
@@ -34,6 +34,10 @@ const std::string kHereTokenEndpointUrl = "here.token.endpoint.url";
 std::string GetDefaultPath() {
   const char* env;
 #ifdef _WIN32
+  if ((env = std::getenv("USERPROFILE")) != nullptr) {
+    return std::string(env).append("\\.here\\credentials.properties");
```

Comment:
It would be nice to have a properties path as a separate string since now it has multiple usages but okay

Reply:
I thought about it but decided that for 2 usages it would not make much sense. Linux path string differs. But if we would have a const var for the Windows case, then we should have a var for the Linux case as well. So it would complicate the code more than the benefits it brings.
For 3 usages I would definitely make a var.